cpp/src/parquet/arrow/arrow_reader_writer_test.cc

```diff
@@ -545,7 +545,7 @@ class TestParquetIO : public ::testing::Test {
     // Also test that slice offsets are respected
     values = values->Slice(5, values->length() - 5);
     std::shared_ptr<ListArray> lists;
-    ASSERT_OK(MakeListArray(values, size, nullable_lists ? null_count : 0,
+    ASSERT_OK(MakeListArray(values, size, nullable_lists ? null_count : 0, "element",
```

Comment:
Hmm... I may be misunderstanding, but how does `ASSERT_OK` work with `MakeListArray` which returns `void`?

Reply:
Confusingly there are two functions named `MakeListArray` with different function signatures. That's irksome, so I'm going to fix it